charts/socks5-server/values.yaml

```diff
@@ -0,0 +1,77 @@
+# socks5-server values file YAML-formatted=
+
+image:
+  repository: "serjs/go-socks5-proxy"
+  # Define this override v{{ .Chart.AppVersion }}, which is the default
+  tag: "v0.0.3"
```

Comment:
Based on the comment above, should this be set to `v{{ .Chart.AppVersion }}` instead?

Reply:
It is optional, but can be removed. I defined it to give it more visibility...